src/content/docs/usage/workspaces.mdx

```diff
@@ -193,148 +194,11 @@ Set the Project Builder at Workspace creation time using command-line
-
-After adding the second project, you will be prompted to add another project to the Workspace.
-
-```text
-Add another project?
-
-[Yes] [No]
-```
-
-Selecting `Yes` will allow you to add additional projects to the Workspace. Once you have added all the projects you need, select `No` to complete the Workspace creation process.
-
-After adding all projects, you will be prompted with a summary of the Workspace configuration.
-
-```text
-SUMMARY
-
-Project #1 - https://github.com/daytonaio/sample-python.git
-   Build       Automatic
-   Image       daytonaio/workspace-project:latest
-   User        daytona
-
-Project #2 - https://github.com/daytonaio/sample-go.git
-   Build       Automatic
-   Image       daytonaio/workspace-project:latest
-   User        daytona
-```
-
-5. Enter a name for your Workspace.
-
-```text
-Workspace name
->
-```
-
-6. Select a Target to deploy your Workspace to. Selecting a Target will prompt you to input the [Target configuration](/docs/configuration/targets). The configurations vary based on the selected Target.
-
-```text
-Choose a Target
-6 items
-===
-local
-docker-provider
-
-Add a Docker Provider Target
-docker-provider (needs installing)
-
-Add a Digital Ocean Provider Target
-digitalocean-provider (needs installing)
-
-Add an AWS Provider Target
-aws-provider (needs installing)
-
-Add a GCP Provider Target
-gcp-provider (needs installing)
-
-Add an Azure Provider Target
-azure-provider (needs installing)
-
-Add a Hetzner Provider Target
-hetzner-provider (needs installing)
-
-Add a Fly.io Provider Target
-fly-provider (needs installing)
-```
-
-Upon Target selection and configuration, Daytona will clone the specified Git repository and set up a new Workspace based on its contents. You will see progress information in the terminal, indicating the process of creating a Workspace.
-
-Once complete, the Workspace will be ready for use with the specified repository and configuration.
-
-```text
-WORKSPACE | ✓ Request submitted
-WORKSPACE | ✓ Creating workspace daytona
-daytona   | Creating project daytona
-daytona   | Pulling image...
-daytona   | Pulling from daytonaio/workspace-project
-...
-daytona   | Project daytona created
-daytona   | Starting project daytona
-daytona   | Project daytona started
-```
-
 ## List Workspaces
 
-Daytona allows you to keep track of your development environments by listing all previously created Workspaces. This helps you manage multiple projects and ensure that you can easily access and maintain your Workspaces.
+Daytona provides an option to keep track of your development environments by listing all previously created Workspaces. This helps you manage multiple projects and ensure that you can easily access and maintain your Workspaces.
 
-1. Run the following command:
+1. Run the following command to list all your Workspaces:
```

Comment:
The list command view was updated. Update the table below

Reply:
![Screenshot 2025-01-22 at 10 29 58](https://github.com/user-attachments/assets/aeb2c088-d780-4f3f-8ca0-8882d2bfa3e1)

This is how it should look like